packages/framework/presence/src/presenceStates.ts

```diff
@@ -23,13 +23,21 @@ export type MapSchemaElement<
 	Keys extends keyof TSchema = keyof TSchema,
 > = ReturnType<TSchema[Keys]>[Part];
 
+/**
+ * @internal
+ */
+export interface LocalUpdateOptions {
+	forceBroadcast: boolean;
+	allowableUpdateLatency?: number;
+}
+
```

Comment:
Let's move this to stateDatastore.ts to avoid cyclical dependencies.
(I also think we can drop forceBroadcast in favor of allowableUpdateLatency to be 0 and mean the same thing.)

I have the API changes that I think make sense for a workspace to have default value. I think it make sense for me to create the same `LocalUpdateOptions` structure.

Reply:
As it turns out it makes sense for there to be two different options interfaces. So this can stay here.
I am naming this one RuntimeLocalUpdateOptions and the one for stateDatastores LocalStateUpdateOptions. ...but also not finalized so I reserved the right to change my mind. :)